src/metalnx-web/src/main/java/com/emc/metalnx/controller/PermissionsController.java

```diff
@@ -110,6 +122,24 @@ public String getPermissionDetails(final Model model, @RequestParam("path") fina
 			throws DataGridException {
 
 		logger.info("Getting permission info for {}", path);
+		
+		String version = irodsServices.findIRodsVersion();
+		
+		int irods_major_version = 4;
+		int irods_minor_version = 3;
+		String[] version_parts = version.split("\\.");
+
+		if (version_parts.length >= 2) {
+			try {
+				irods_major_version = Integer.parseInt(version_parts[0]);
+				irods_minor_version = Integer.parseInt(version_parts[1]);
+			} catch (NumberFormatException e) {
+				// just leave the default
+			}
+		}
+		
+		boolean irods_43_or_greater = irods_major_version > 4 || 
+				(irods_major_version == 4 && irods_minor_version >= 3);
```

Comment:
Hmm, the minor version should probably default to 2 in case L135 fails.

This isn't a blocker since I'm guessing the call to `irodsServices.findIRodsVersion()` never fails as long as it comes from the iRODS server.

Let's open an issue to revisit this after the release.

Reply:
Why would we want to default to 4.2?

Reply:
I wrote issue #381 for this.

Reply:
> Why would we want to default to 4.2?

Falling back to 4.2 could very well be the wrong decision, but in the case where there are users still running 4.2.12 and want to run Metalnx 3 against it, this would allow them to do so. Otherwise, Metalnx 3 is effectively requiring a 4.3 server.

Having that issue allows us to think it through more rather than having to make a decision now.